pkg/handler/storage/client.go

```diff
@@ -66,9 +68,15 @@ func convertV2(in *regionv1.FileStorage) *openapi.StorageV2Read {
 	return &openapi.StorageV2Read{
 		Metadata: conversion.ProjectScopedResourceReadMetadata(in, in.Spec.Tags),
 		Spec: openapi.StorageV2Spec{
-			Attachments: &openapi.StorageAttachmentV2Spec{},
-			StorageType: openapi.StorageTypeV2Spec{},
-			Size:        in.Spec.Size.String(),
+			Attachments: &openapi.StorageAttachmentV2Spec{
+				NetworkIDs: convertAttachmentsList(in.Spec.Attachments),
+			},
+			StorageType: openapi.StorageTypeV2Spec{
+				NFS: &openapi.NFSV2Spec{
+					RootSquash: in.Spec.NFS.RootSquash,
+				},
+			},
+			Size: in.Spec.Size.String(),
```

Comment:
This takes the `resource.Quantity` to a string; assuming we are OK with whichever format it comes with, :+1:

Reply:
Briefly. I don't think `SizeGiB int` is unreasonable, but I think `resource.Quantity` is better, so you can supply e.g., `1.5Ti`.